docs/proposals/compute-resource-metrics-api.md

```diff
@@ -35,15 +35,31 @@ Documentation for other releases can be found at
 
 ## Goals
 
-Provide resource usage metrics on pods and nodes on the API server to be used
-by the scheduler to improve job placement, utilization, etc. and by end users
-to understand the resource utilization of their jobs. Horizontal and vertical
-auto-scaling are also near-term uses.
+Provide resource usage metrics on pods and nodes through the API server to be
+used by the scheduler to improve job placement, utilization, etc. and by end
+users to understand the resource utilization of their jobs. Horizontal and
+vertical auto-scaling are also near-term uses. Additionally, a subset of the
```

Comment:
Yeah. Multiple commits followed by a squash after LGTM is the recommended
approach for PRs that have lots of comments.

On Thu, Oct 22, 2015 at 5:35 PM, Tim St. Clair <notifications@github.com>
wrote:

> In docs/proposals/compute-resource-metrics-api.md
> <https://github.com/kubernetes/kubernetes/pull/15862#discussion_r42823261>
> :
>
> > @@ -35,15 +35,31 @@ Documentation for other releases can be found at
> >
> >  ## Goals
> >
> > -Provide resource usage metrics on pods and nodes on the API server to be used
> > -by the scheduler to improve job placement, utilization, etc. and by end users
> > -to understand the resource utilization of their jobs. Horizontal and vertical
> > -auto-scaling are also near-term uses.
> > +Provide resource usage metrics on pods and nodes through the API server to be
> > +used by the scheduler to improve job placement, utilization, etc. and by end
> > +users to understand the resource utilization of their jobs. Horizontal and
> > +vertical auto-scaling are also near-term uses. Additionally, a subset of the
>
> Ok, works for me.
>
> For future reference, if I make changes in separate commits (don't squash)
> does it prevent github from collapsing comment threads?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/15862/files#r42823261>.
>